introduction.md

```diff
@@ -1,2 +1,9 @@
 Get back in level 1 b'cause i miss something
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?